src/GAP/iso_oscar_gap.jl

```diff
@@ -55,12 +55,17 @@ function _iso_oscar_gap(RO::Union{Nemo.zzModRing, Nemo.ZZModRing})
    return MapFromFunc(RO, RG, f, finv)
 end
 
-# Assume that `FO` and `FG` are finite fields of the same order
-# in Oscar and GAP, respectively.
-function _iso_oscar_gap_field_finite_functions(FO::Union{Nemo.fpField, Nemo.FpField}, FG::GAP.GapObj)
+_ffe_to_int(a::FqFieldElem) = Nemo._coeff(a, 0)
+_ffe_to_int(a::FqPolyRepFieldElem) = coeff(a, 0)
```

Comment:
```suggestion
_ffe_to_int(a::FqPolyRepFieldElem) = coeff(a, 0)
_ffe_to_int(a::fqPolyRepFieldElem) = ZZ(coeff(a, 0))
```
(There are tests showing that a method for `fqPolyRepFieldElem` is missing.
The tests intended to check `fqPolyRepField` and `fqPolyRepFieldElem` did not exhibit this because meanwhile `GF(p, 2)` returns a `FqField`.)